docs/programming/parallel-programming-fundamentals.md

```diff
@@ -16,7 +16,7 @@ weekend it takes you several weeks to finish the puzzle. Now assume
 you have a team of friends helping with the puzzle. It progresses much faster,
```

Comment:
Please make sure to go through this page (and other pages) and make sure to highlight items that should have code syntax and correct those that are wrong. For example, 
![Screenshot 2024-09-18 at 4 54 31 PM](https://github.com/user-attachments/assets/d0cf8286-0175-421e-ad9c-bf36104c8931)